examples/iterative_probabilistic_kh.py

```diff
@@ -85,7 +100,17 @@ def iterative_refine_experiment(
     t_schedule: Optional[Array] = None,
     seed: int = 0,
 ) -> tuple[Array, Coresubset]:
-    """Perform an experiment with n_iter iterations and a given temperature schedule."""
+    """
+    Perform an experiment by iteratively refining the coreset using Kernel Herding.
+
+    :param data: A :class:`Data` instance
```

Comment:
Should be more specific - the type is already given by the type hint above. Something more like this, perhaps?
```suggestion
    :param data: The dataset to iteratively refine a coreset from
```